### PR TITLE
Fixed pick up

### LINKED ISSUE #58: Pickup is missing walking
As is, the pickup task, runs for 1 tick while you are forced to stand still.

In regular factorio, 
- it is something you can do while walking/mining/shooting etc. 
- it is something you do for awhile

### PR DESCRIPTION
**Pull request by MortenTobiasNielsen**
I couldn't see a way to make it backwards compatible due to how wrong I had implemented it. I hope that a run can be re-aligned with a combination of walking, picking and idle tasks. 

closes #58

### CODE REVIEW

File: Lua Files/control.lua [hunk 1]
```diff
@@ -856,13 +857,14 @@ local function doStep(steps)
 		return drop()
 
 	elseif steps[2] == "pick" then
+		pick = steps[3] - 1
```
Comment (theis999) [inline Lua Files/control.lua:860]: Maybe `pick = pick + steps[3] - 1` in case of two of pick up task in sequence.

File: Lua Files/control.lua [hunk 2]
```diff
@@ -8,6 +8,7 @@ local run = true
 local step = 1
 local step_reached = 0
 local idle = 0
+local pick = 0
```
Comment (theis999) [inline Lua Files/control.lua:11]: I would rather have a more qualified named, like pickup_ticks.

**Review by theis999 — APPROVED** [review]
Looks like a solid conversion to a much more correct implementation 👍

File: Lua Files/control.lua (hunk 1 above)
Comment (MortenTobiasNielsen) [reply Lua Files/control.lua:860]: Agreed.

File: Lua Files/control.lua (hunk 2 above)
Comment (MortenTobiasNielsen) [reply Lua Files/control.lua:11]: Agreed.

**Pull request merged**